schema/RequestSignInLinkMutation.re

```diff
@@ -18,7 +21,50 @@ let requestSignInLink =
       "requestSignInLink",
       ~typ=non_null(GraphqlTypes.errorPayloadType),
       ~args=Arg.[arg("input", ~typ=non_null(requestSignInLinkInput))],
-      ~resolve=(_, (), input) =>
-      Lwt.return(Ok(None))
+      ~resolve=(context: Context.t, (), email) => {
+        let%lwt user = User.getByEmail(context.connection, email);
+        switch (user) {
+        | Some(user) =>
+          let%lwt (response, _) =
+            Sendgrid.sendEmail(
+              ~to_=email,
+              ~subject="Summhub - Login Access Link",
+              ~content=
+                "<a href=\"https://summhub.com/sign-in/\""
```

Comment:
nice just need to move to User and put the "site" as a constant somewhere, maybe a Constants.re in Util

Reply:
hmm top, gonna do this